config/mongo.js

```diff
@@ -17,11 +18,7 @@ const connectMongo = async () => {
   }
 
   try {
-    const options = {
-      autoIndex: true,
-      family: 4,
-      maxPoolSize: 10
-    };
+    const options = { autoIndex: true, family: 4 };
```

Comment:
We need to keep `maxPoolSize` otherwise with serverless we hit the Mongo Atlas connection limit of 1.5k

Reply:
sorry, I don't know what happened here. I didn't touch that code.